app/helpers/moments_form_helper.rb

```diff
@@ -82,13 +82,23 @@ def moment_publishing
       checked: !@moment.published? }
   end
 
+  def moment_bookmarked
+    moment_input_props('bookmarked', 'switch', 'moments.bookmark')
```

Comment:
I think we'll want to hide this behind a feature flag since it's part of a broader feature. That way it doesn't show up in production. I would just add these prop when `Rails.env.development?` is true.

Reply:
Sure!